### ISSUE TITLE
[Link and button labels] Button and link language isn't descriptive and/or their purpose isn't clear. (08.03.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#34875]
#### VFS team name
Claims & Appeals

#### VFS product name
HLR V2

#### Point of Contact/Reviewers
Brian DeConinck (@briandeconinck) - Accessibility

---

### Platform Issue
Button and link language isn't descriptive and/or their purpose isn't clear.

### Issue Details
Step 1 contact information, "Edit" links don't have visibly unique names. Although each one has unique name for screen reader users using class="sr-only", sighted voice command users (eg. Dragon NaturallySpeaking) won't see the sr-only text and won't know what unique name to speak.

### VA.gov Experience Standard
[Category Number 08, Issue Number 03](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 2.4.4_A

---

### Platform Recommendation
Make the sr-only text visible, eg. "Edit mobile phone number"

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] Close the ticket when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment in the ticket.

**Comment by Mottie** [comment]
I would like to discuss this further with @briandeconinck & @kevinstachura. The space of the issue card is limited and including all of the text from the `sr-only` content would make the edit button too wide to fit the design.

**Comment by Mottie** [comment]
The `aria-label` fix has been merged, and will be available in staging soon.

**Comment by briandeconinck** [comment]
Hi @Mottie! Sorry for the slow follow-up on my end. Unfortunately, putting the text in an `aria-label` doesn't resolve this issue.

`aria-label` and `sr-only` text provide extra text to be announced by screen readers. The situation I'm thinking of is for users of voice command software, eg. someone unable to use a keyboard/mouse. These users frequently won't be screen reader users and are operating their computer via voice commands based on what's visible to them on screen.

For voice command software like Dragon NaturallySpeaking, there are standard commands like `Click <link name>`. When the text of a link is visibly unique, users can speak the full link text and follow them immediately. When there are name collisions, users have extra steps to differentiate and select the one they wanted.

But I definitely hear you on the design constraints. Since I'm brand new and I don't want to send you down this road for no reason, I'm going to chat with Noah and get back to you.

**Comment by briandeconinck** [comment]
Hi @Mottie! Checked in with Noah, and the links as currently coded are fine and consistent with past guidance given by this group. Since there are only a few Edit links and they're easy to distinguish visually, the impact for Dragon users will be fairly small.

You're good to go! Thanks for bearing with me while I settle in!

**Comment by Mottie** [comment]
Ok, thank you @briandeconinck!
